.memento/scripts/ticket-context.sh

```diff
@@ -1,9 +1,28 @@
 #!/bin/sh
 if [ -z "$1" ]; then
+  echo "## Tickets Overview"
+  echo ""
   npx memento-protocol ticket list 2>/dev/null || echo "No tickets found"
+  echo ""
+  echo "**To access ticket content: Use Read tool with file paths like:**"
+  echo "- .memento/tickets/next/ticket-name.md"
+  echo "- .memento/tickets/in-progress/ticket-name.md"
+  echo "- .memento/tickets/done/ticket-name.md"
 else
-  find .memento/tickets -name "*$1*" -type f | head -1 | xargs cat 2>/dev/null || {
```

Comment:
The command when users input a ticket name should dump the entire ticket context into context. No need for extra data here